components/did/Core/Src/DID.c

```diff
@@ -115,6 +116,14 @@ void parse_can_message( uint8_t* CAN_rx_data, uint32_t CAN_ID )
 			SET_CYCLIC_DATA( data_MCB_drive_state, CAN_rx_data[0] );
 			break;
 
+		/*
+		 *  MCB Diagnostics
+		 */
+		case CAN_ID_MCB_REGEN_ENABLED:
+			uint8_t regen_enabled = GETBIT(CAN_rx_data[3], 1);
+			SET_CYCLIC_DATA( data_MCB_regen_enabled, regen_enabled);
+			break;
+
```

Comment:
The current MCB code shows that it is data byte 2 not 3?:
![image](https://github.com/UBC-Solar/firmware_v3/assets/117491745/eb4a38fb-4f7f-474b-8750-9e4edd0b19d8)